documentation/tutorials/deploy-kernel.rst

```diff
@@ -0,0 +1,179 @@
+===============================
+  How to Install a New Kernel
+===============================
+.. _deploy-kernel:
+
+.. contents::
+   :depth: 1
+   :local:
+   :backlinks: none
+
+.. highlight:: console
+
+Deploy Overview
+---------------
+
+Alright, at this point, we expect that you have already learned how to compile
+the :ref:`Linux kernel from the source<buildlinux>` and :ref:`manage your config file<configm-tutorial>`
+as described in the previous tutorials (if not, please, read those tutorials
+first). Now that you know how to compile your new kernel, you probably want to
+know how to install it to a Linux-based system. Don't worry, kw is here to
+rescue you.
+
+One of the most important features associated with kw is managing the
+deployment of new kernels to a target machine. By a target machine, we have
+three types:
+
+1. Remote: Any machine you can access via ssh, with root permission.
+2. Local: A machine that you have direct access to, such as your laptop or
+   desktop.
+3. Virtual Machine (VM): A QEMU virtual machine to which you can access the
+   image file.
+
+Don't worry; you will learn how to handle each of these targets.
+
+Kw Deploy Limitations
+---------------------
+
+Let's start by setting the expectations around kw deploy and describing its
+current limitations:
+
+* The target machine must be Debian or ArchLinux family (by family, we are
+  talking about derivative distros). For example, Ubuntu and Mint are derivated
+  from Debian, meaning that kw supports those distros.
+* Right now, kw only supports GRUB2 bootloader.
+
+Notice that these limitations can change in the future; consider to help us to
+improve this in the future.
+
+Tutorial Assumptions
+--------------------
+
+Now that you know kw limitations, we also assume the following tasks are
+completed:
+
+* You already compiled your kernel 
+* You have a functional `.config` file
+* For remote deploy, you have root access to the target machine.
```

Comment:
I don't remember, but what does "root access" comprise? For example, ssh key, passwordless access?

Reply:
I understand that you answer it in the `kw ssh` section below. Maybe point out the section is enough.